src/main/java/uk/gov/hmcts/ccd/domain/service/aggregated/SearchQueryOperation.java

```diff
@@ -64,7 +76,52 @@ public SearchResultView execute(final String view,
             }
         }
         draftsAndCases.addAll(cases);
-        return mergeDataToSearchResultOperation.execute(caseType.get(), draftsAndCases, view, draftResultError);
+        return mergeDataToSearchResultOperation.execute(caseType.get(), searchResult, draftsAndCases, draftResultError);
+    }
+
+    private SearchResult getSearchResult(final CaseType caseType, final String view) {
```

Comment:
mind calling this `getSearchResultDefinition` ?

Reply:
done